dotnet/example_code/DynamoDB/CreateTablesLoadData.cs

```diff
@@ -1,13 +1,13 @@
-// snippet-sourcedescription:[CreateTablesLoadData.cs demonstrates how to create a DynamoDB table ]
+// snippet-sourcedescription:[CreateTablesLoadData.cs demonstrates how to ]
 // snippet-service:[dynamodb]
 // snippet-keyword:[dotNET]
 // snippet-keyword:[Amazon DynamoDB]
 // snippet-keyword:[Code Sample]
-// snippet-keyword:[CreateTable, DeleteTable]
+// snippet-keyword:[ ]
 // snippet-sourcetype:[full-example]
 // snippet-sourcedate:[ ]
 // snippet-sourceauthor:[AWS]
-// snippet-start:[dynamodb.dotNET.CodeExample.CreateTable]
+// snippet-start:[dynamodb.dotNET.CodeExample.a452f749-fd31-42f8-9a8d-7acd2c6f4944] 
```

Comment:
This code sample id is being used in another guide and should be be removed. 
You can add an id if you want, but please do not remove any.

Reply:
that file was a test to make sure everything worked end-end